### ISSUE TITLE
lsDS not compatible with DSLite

### ISSUE THREAD

**Issue by ymarcon** [datashield/dsBase#280]
Because it calls for `.GlobalEnv` in place of  `parent.frame()`:
https://github.com/datashield/dsBase/blob/master/R/lsDS.R#L40

**Comment by StuartWheater** [comment]
Yes, it is on my "to do" list to ensure dsBase is fully comparable with DSLite. I plan to using dsBaseClient test suit to help with this, but will need to ensure all the test datasets are supported.
